drivers/soundwire/cadence_master.c

```diff
@@ -1278,6 +1282,53 @@ bool sdw_cdns_check_resume_status(struct sdw_cdns *cdns)
 }
 EXPORT_SYMBOL(sdw_cdns_check_resume_status);
 
+/**
+ * sdw_cdns_resume: Cadence PM resume routine
+ *
+ * @cdns: Cadence instance
+ */
+int sdw_cdns_resume(struct sdw_cdns *cdns)
+{
+	int ret;
+
+	/* Disable block wakeup */
+	cdns_updatel(cdns, CDNS_MCP_CONTROL, CDNS_MCP_CONTROL_BLOCK_WAKEUP, 0);
```

Comment:
why is this here? It's already set on suspend?

Reply:
yes, it is set in suspend.  Comments should be refined.